src/more-infos/more-info-updater.html

```diff
@@ -0,0 +1,55 @@
+<link rel="import" href="../../bower_components/polymer/polymer.html">
+<link rel="import" href="../../bower_components/paper-button/paper-button.html">
+
+<dom-module id="more-info-updater">
+  <style>
+    .data-entry .value {
+      max-width: 200px;
+    }
+    .center {
+      text-align: center;
+    }
+    .centered {
+      margin: auto;
+    }
+    .divider {
+      padding: 5px;
+    }
+    .link {
+      color: #03A9F4;
+    }
+  </style>
+  <template>
+    <div class='layout vertical'>
+
+      <!-- GitHub Link -->
+      <div class='data-entry layout justified horizontal center' style="width:100%" id='setting'>
+        <paper-button class='centered link' on-tap="linkTapped">
```

Comment:
If you just want to render a link, no need to use a Polymer layout but just use old skool HTML:

```html
<a href='[[stateObj.attributes.link]]' target='_blank'>View on GitHub</a>
```